Golem/Yagna/types.cs

```diff
@@ -43,7 +43,6 @@ private PaymentDriver(string id)
         }
 
         public static readonly PaymentDriver ERC20 = new PaymentDriver("erc20");
-        public static readonly PaymentDriver ERC20next = new PaymentDriver("erc20next");
```

Comment:
Yagna `pre-rel-v0.16.0-ai-rc7` no longer comes with `erc20next` so I removed it.